CWE-94-Python_darwin/mit-han-lab_neurips-micronet/u.py

```
@@ -0,0 +1,944 @@
+        f.write(string)
+
+def load_pickle(path):
+    with open(path, 'rb') as f:
+        return pickle.load(f)
+
+def save_pickle(path, obj):
+    with open(path, 'wb') as f:
+        pickle.dump(obj, f)
+
+def wget(link, output_dir):
+    cmd = 'wget %s -P %s' % (path, output_dir)
```

Ruff (0.13.2)

Undefined name `path`

(F821)